### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by sjmulder**
Add pkgsrc link and some minor tweaks. If you don't like the changes feel free to just add the pkgsrc link or something, no point in bikeshedding.

Ninja edit: at time of writing pkgsrc.se had not yet indexed the new package but it soon will.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Noice is Not Noice, a noicer fork...
 
 [Integrate utilities](https://github.com/jarun/nnn#sample-scripts) like sxiv or fzy easily, transfer selected files using lftp or use it as a [(neo)vim plugin](https://github.com/jarun/nnn#neovim-plugin); `nnn` supports as many scripts as you need!
 
-It runs on Linux, macOS, Raspberry Pi, Cygwin, Linux subsystem for Windows and Termux.
+It runs on Linux, macOS, BSD, Cygwin, Linux subsystem for Windows, Termux, and more.
```
Comment (jarun) [inline README.md]: Please state:

`It runs on Linux, macOS, BSD, Cygwin, Linux subsystem for Windows and Termux.`

_more_ is undefined. Some unhappy users might hate us for this. ;)

File: README.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ Intrigued? Find out [HOW](https://github.com/jarun/nnn/wiki/performance-factors)
 
 #### Dependencies
 
-`nnn` needs libncursesw (on Linux or ncurses on macOS) and standard libc.
+`nnn` needs curses with wide character support (like ncursesw) and standard libc.
```
Comment (jarun) [inline README.md]: a curses library with...

**Comment by jarun** [comment]
I have completed the review. Please check.

File: README.md (hunk 1 above)
Comment (sjmulder) [reply README.md]: Tried to build on Illumios and Haiku to make a point but both required some tweaks, so guess you've right, ha.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**